activerecord/README.rdoc

```diff
@@ -195,6 +195,35 @@ Source code can be downloaded as part of the Rails project on GitHub:
 * https://github.com/rails/rails/tree/master/activerecord
 
 
+== Getting Started
+
+1.install sqlite to your PC
```

Comment:
Can you update these to have a space after the period and then make it capitalized?

Example: "1. Install SQLite on your computer"